### ISSUE TITLE
Trouble following packaging libraries tutorial

### ISSUE THREAD

**Issue by ljhopkins2** [pypa/packaging-problems#293]
Thank you for providing feedback on Python packaging!

To help us help you, please fill out as much of the following as you can. If a question is not relevant, feel free to skip it.

1. What is your operating system and version?

- `MacOS Mojave 10.14.6 (18G103)`

2. What is your Python version?

- `3.7.5`

3. What version of pip do you have?

- `pip 19.3.1 from /anaconda3/envs/bart_virtenv/lib/python3.7/site-packages/pip (python 3.7)`

4. If following an online tutorial or guide, please provide a link to the page or section giving you trouble:

https://packaging.python.org/tutorials/packaging-projects/#installing-your-newly-uploaded-package

5. Could you describe your issue in as much detail as possible?

_NOTE: I'm using conda's virtual environment b/c I couldn't get venv to work (activating kept pointing to the base environment)._

Everything seems to work great until I try running `import example_pkg`:
```python
(bart_virtenv) user-Air:~ user$ python
Python 3.7.5 (default, Oct 25 2019, 10:52:18) 
[Clang 4.0.1 (tags/RELEASE_401/final)] :: Anaconda, Inc. on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import example_pkg
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ModuleNotFoundError: No module named 'example_pkg'
```

In `anaconda3/envs/bart_virtenv/lib/python3.7/site-packages`, I have a folder called `example_pkg_ljhopkins2-0.0.1.dist-info` with files named `INSTALLER`, `LICENSE`, `METADATA`, `RECORD`, `top_level.txt`, and `WHEEL`.

Got this message when I tried installing a second time for good measure:

```
Requirement already satisfied: example-pkg-ljhopkins2 in /anaconda3/envs/bart_virtenv/lib/python3.7/site-packages (0.0.1)
```

**Comment by merwok** [comment]
Do you see `example_pkg` package (directory with `__init__.py` file) in site-packages?  If not, you have the packaging metadata there but not the package code itself.

I tried to find your package on PyPI to look but didn’t succeed.
Are you installing from local source?  If so, can you show your setup.py and/or setup.cfg.  You need a `packages` option defined in one of these.

**Comment by ljhopkins2** [comment]
Hey Éric,

I don't see an `example_pkg` directory. Package is at https://test.pypi.org/project/example-pkg-ljhopkins2/.

Here's my `setup.py` (I copy/pasted from the tutorial):
```
import setuptools
with open("README.md", "r") as fh:    
    long_description = fh.read()
 
setuptools.setup(
        name="example-pkg-ljhopkins2", # Replace with your own username
        version="0.0.1",    author="Example Author",
        author_email="author@example.com",
        description="A small example package",
        long_description=long_description,
        long_description_content_type="text/markdown",    
        url="https://github.com/pypa/sampleproject",
        packages=setuptools.find_packages(),
        classifiers=[
            "Programming Language :: Python :: 3",
            "License :: OSI Approved :: MIT License",
            "Operating System :: OS Independent",
        ],
       python_requires='>=3.6',)
```

Thanks for your help.

**Comment by merwok** [comment]
If you compare your local files to https://packaging.python.org/tutorials/packaging-projects/#creating-the-package-files , is there a difference?

**Comment by ljhopkins2** [comment]
Yes, I had my directory structure wrong, with the `__init__.py` file in the same directory as `setup.py` et. al. when I ran `sdist`. I corrected that and re-uploaded to test as v0.0.2 (https://test.pypi.org/project/example-pkg-ljhopkins2/).

But I'm still getting a `ModuleNotFoundError` when I `import example_pkg`. Here are the contents of `anaconda3/.../site-packages` after uninstalling and reinstalling via `pip`:

![image](https://user-images.githubusercontent.com/49728392/68224969-a1cd5100-ffbd-11e9-8a7a-8aa0cf029a4e.png)
